src/components/modal-window/index.jsx

```diff
@@ -1,92 +1,153 @@
+              <label className="modal-label">
+                Title
+                <input
+                  type="text"
+                  name="title"
+                  placeholder="Title"
+                  value={movie.title}
+                  onChange={setTitle}
+                />
+              </label>
+              <label className="modal-label-second">
+                RELEASE DATE
+                <input
+                  type="date"
+                  name="releseDate"
+                  placeholder="yyyy-mm-dd"
+                  value={movie.relese_date}
+                  onChange={setReleseDate}
+                />
+              </label>
+            </fieldset>
+            <fieldset className="modal-fieldset">
+              <label className="modal-label">
+                movie url
+                <input
+                  type="url"
+                  name="movieUrl"
+                  placeholder="https://"
+                  value={movie.poster_path}
+                  onChange={setMovieUrl}
+                />
+              </label>
+              <label className="modal-label-second">
+                RATING
+                <input
+                  type="number"
+                  name="rating"
+                  placeholder="7.8"
+                  value={movie.vote_average}
+                  onChange={setRating}
+                />
+              </label>
+            </fieldset>
+            <fieldset className="modal-fieldset">
+              <label className="modal-label">
+                genre
+                <select multiple>
+                  <option value="crime" defaultValue>
+                    crime
+                  </option>
+                  <option value="horror" defaultValue>
+                    horror
+                  </option>
+                </select>
+              </label>
+              <label className="modal-label-second">
+                RUNTIME
+                <input
+                  type="number"
+                  name="runtime"
+                  placeholder="minutes"
+                  value={movie.runtime}
+                  onChange={setRuntime}
+                />
+              </label>
+            </fieldset>
+            <fieldset className="modal-fieldset">
+              <label className="modal-label">
+                OVERVIEW
+                <textarea
+                  className="overview-input"
+                  type="text"
+                  name="overview"
+                  placeholder="Movie description"
+                  value={movie.overview}
+                  onChange={setOverview}
+                />
+              </label>
+            </fieldset>
+            <p className="modal-buttons">
+              <input className="modal-reset" type="reset" value="reset" />
+              <input className="modal-submit" type="submit" value="submit" />
+            </p>
+          </form>
+        </div>
+      </div>
+    </>
+  );
+};
+
```

Comment:
It's better to think about input types from a UX/UI point of view.
Offtop: did you ever think about input type='number' for phone numbers? 😃